libnl/nl80211/iw_scan.py

```diff
@@ -504,7 +504,14 @@ def get_secchan_offs(type_, data):
     type_ -- corresponding `ieprinters` dictionary key for the instance.
     data -- bytearray data to read.
     """
-    raise NotImplementedError
+    chans = [
```

Comment:
This list is already defined as the module variable `ht_secondary_offset` at the top of this file on line 67. Please use that instead.